### PR TITLE
NULL support bindings

### PR DESCRIPTION
**Pull request by gsvic**

### CODE REVIEW

**Comment by joe-maley** [comment]
@gsvic I will take a look at this once the core PR is merged, hopefully next week.

File: gradle.properties
Comment (joe-maley) [inline gradle.properties:2]: Should we wait until we release 2.2 with null support?

File: src/test/java/io/tiledb/java/api/AttributeTest.java
Comment (joe-maley) [inline src/test/java/io/tiledb/java/api/AttributeTest.java:116]: there appears to be an indentation issue here

**Review by joe-maley — COMMENTED** [review]
LGTM, two minor comments:

File: gradle.properties
Comment (gsvic) [reply gradle.properties:2]: Yes, I agree. I changed the tag to `dev` just for the CI. So let's wait for the 2.2 release.

File: src/test/java/io/tiledb/java/api/AttributeTest.java
Comment (gsvic) [reply src/test/java/io/tiledb/java/api/AttributeTest.java:116]: I think that this is ok. If you look at it, the `Attribute a = ..` is inside the `try` parentheses (`try (Context ctx = ... Attribute a = ...)` similarly to an `if () {}` block. So that's why the `a.setNullable` (line 116) is positioned two tabs (`\t`) after the `try` (line 114).

What do you think?

**Review by Shelnutt2 — APPROVED** [review]

**Pull request merged**
